test/failure_handler/src/share/classes/jdk/test/failurehandler/ToolKit.java

```diff
@@ -51,6 +56,26 @@ public void gatherEnvironmentInfo(HtmlSection section) {
         }
     }
 
+    @Override
+    public void gatherCoreInfo(HtmlSection section, Path core) {
+        if (core.getFileName().toString().endsWith(".gz")) {
+            Path unpackedCore = Path.of(core.toString().replace(".gz", ""));
+            try (GZIPInputStream gzis = new GZIPInputStream(Files.newInputStream(core))) {
+                Files.copy(gzis, unpackedCore);
+                for (ActionSet set : actions) {
+                    set.gatherCoreInfo(section, unpackedCore);
+                }
+            } catch (IOException ioe) {
+                // just silently skip gzipped core processing
```

Comment:
we don't silently ignore exceptions in `failure_handler`, we tend to print them so we at least have some echoes of what happened.